pom.xml

```diff
@@ -43,8 +43,6 @@
     <!-- Define JaCoCo agent argLine property to avoid Surefire failure when JaCoCo is not enabled. -->
     <jacoco.agent.argLine/>
 
-    <maven.compiler.release>11</maven.compiler.release>
```

Comment:
This is the interesting one: the `jboss-parent:37` upgrades the `maven-compiler-plugin` to `3.8.1-jboss-1`, which for some reason clashes with this property in the `test-compile` phase.